cypress/e2e/pages/onDemandAudio/index.cy.js

```diff
@@ -1,32 +1,487 @@
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/bbc_nepali_radio/w172xzcfvptk838', // On Demand Episode
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/podcasts/p02pc9w3/p09j0dm5', // Podcast Episode
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/podcasts/p02pc9w3', // Podcast Brand
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/programmes/p0340yr4', // On Demand Brand
+    service: 'pashto',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/programmes/p0340ysc', // On Demand Brand
+    service: 'pashto',
+    runforEnv: 'test',
+    tests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/w3ct26m6', // On Demand Episode
+    service: 'pashto',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/persian/bbc_dari_radio/programmes/p0340v0s', // On Demand Brand Dari
+    service: 'persian',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/persian/bbc_dari_radio/programmes/p0364sj5', // On Demand Brand Dari
+    service: 'persian',
+    runforEnv: 'test',
+    tests,
+  },
+  {
+    path: '/persian/bbc_dari_radio/w172y2n5p9pfj6x', // On Demand Episode Dari
+    service: 'persian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/persian/podcasts/p02pc9wf', // Podcast Brand
+    service: 'persian',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/persian/podcasts/p02pc9wf/p09knl1v', // Podcast Episode
+    service: 'persian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/portuguese/podcasts/p07r3r3t', // Podcast Brand
+    service: 'portuguese',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/portuguese/podcasts/p07r3r3t/p09clrrg', // Podcast Episode
+    service: 'portuguese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/russian/podcasts/p05607v8/p06x0tn3', // Podcast Episode
+    service: 'russian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/russian/podcasts/p05607v8', // Podcast Brand
+    service: 'russian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/somali/bbc_somali_radio/programmes/p034117j', // On Demand Brand
+    service: 'somali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/somali/bbc_somali_radio/w172xzzpp131z23', // On Demand Episode
+    service: 'somali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/swahili/bbc_swahili_radio/programmes/p03411mj', // On Demand Brand
+    service: 'swahili',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/swahili/bbc_swahili_radio/programmes/p030s7gh', // On Demand Brand
+    service: 'swahili',
+    runforEnv: ['test'],
+    tests,
+  },
```

Comment:
Because test points to live data, we can pick one path and combine them into a single entry, with runforEnv: ['test', 'live']